setup/install/providers/kubernetes-v2/index.md

```diff
@@ -42,6 +42,75 @@ The Kubernetes provider has two requirements:
     API resource that `kubectl` supports is also supported by Spinnaker. This
     is an improvement over the original Kubernetes provider in Spinnaker.
 
+
+<span class="begin-collapsible-section"></span>
+
+### Optional: Create a Kubernetes Service Account
+
+If you want, you can associate Spinnaker with a [Kubernetes Service
+Account](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/),
+even when managing multiple Kubernetes clusters. This can be useful if you need
+to grant Spinnaker certain roles in the cluster later on, or you typically
+depend on an authentication mechanism that doesn't work in all environments.
+
+Given that you want to create a Service Account in context `$CONTEXT`, create
+the following resources with `kubectl apply --context $CONTEXT ...`
+
+```yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spinnaker
+  namespace: spinnaker
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: spinnaker-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: spinnaker-service-account
+  namespace: spinnaker
+```
+
+> Note, this grants Spinnaker full access to the Kubernetes cluster using the
+> `cluster-admin` role binding.
```

Comment:
Can we create a k8s service account with a role other than `cluster-admin`? 
IOW, is this sentence just for this *example*, or do we always give Spinnaker full access when creating a K8s serivce account?

Reply:
Good point, this is just an example. I will clarify.